BoardWebApp-Tests/LoginValidationTests.cs

```diff
@@ -42,26 +43,12 @@ public void PostLoginDisplaysErrorMessage(string email, string password, bool ex
                 }
             };
 
-            _output.WriteLine("form data - email: " + loginInfo.userLoginModel.Email);
-
             var result = new AccountController(_dbContext).Login(loginInfo);
+            var requestResult = Assert.IsType<ViewResult>(result);
+            var resultModel = Assert.IsAssignableFrom<SendLoginModel>(requestResult.ViewData.Model);
```

Comment:
what does IsAssignableFrom do?

Reply:
I believe it makes sure that whatever is in the brackets will fit in a SendLoginModel variable.
e.g. a SendLoginModel instance or a sub-class (we don't have sub-classes in our case)